src/components/DeleteComment.jsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { deleteCommentById } from "../api";
+
+const DeleteComment = ({ comment_id, setComments }) => {
+  const deleteSingleComment = () => {
+    let feedbackMsg = document.getElementById("comment-delete-message");
```

Comment:
we should never need to work on the dom directly like this. We could instead have something that holds this message that then gets displayed after the comment is deleted 🙂